src/main/java/org/concordion/internal/util/Announcer.java

```diff
@@ -9,10 +9,12 @@
 import java.util.List;
 
 /**
+ * @deprecated Depricated due to performance issues - use java.util.List instead

```

Comment:
Please add a bit more detail on the performance issue. I'd suggest adding something like:
`When cross-compiled to Concordion.NET, the Announcers generate dynamic proxies that are realized by code compiled at run time. When profiled, the Announcer took 1460ms, vs 5ms for the list based solution.`